zest/releaser/utils.py

```diff
@@ -2,11 +2,11 @@
 
 from argparse import ArgumentParser
 from colorama import Fore
-from pkg_resources import parse_version
+from packaging import version
```

Comment:
`version` is probably used here and there as a variable name. I'm going to change this import to `from packaging.version import parse as parse_version`. I'll do some branch/merge juggling :-)